docs/changelog.md

```diff
@@ -3,6 +3,15 @@ id: changelog
 title: Changelog
 sidebar_label: Changelog
 ---
+### Major features and improvements
+
+- `--python-submodules-directory` flag is added with the default value set to the *mage*. It points to the directory where the Python's utility submodules are stored.
+
+### Bug Fixes
+- The `EmptyResult` sink operator was added to the Memgraph's planner. This means that results produced by a query `MATCH (n) SET n.test_prop = 2` will get exhausted which was a problem in some Bolt clients implementations, e.g in Golang's client.
```

Comment:
why is this a bugfix, not a feature?

Reply:
it is a bug from the user's perspective, one developer opened an issue.